### DISCUSSION TITLE
Unable to link my google account

### DISCUSSION THREAD

**Discussion by ghost** [ReVanced/revanced-discussions#875] [Question] [answered]
Recently my revanced stopped loading so I cleared data of microg. Now the microg account is not getting linked.It says "Please check your connection" everytime.

![Screenshot_20230121_081114](https://user-images.githubusercontent.com/7278960/213840335-f4543a45-7332-4c89-ac77-48fcb557bf35.jpg)

**Comment by GeekCornerGH** [comment, marked as answer]
Try removing microg account from your phone settings and add it back
